tests/config/main.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 The clvk authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdlib>
+#include <fstream>
+#include <gtest/gtest.h>
+
+#ifdef CLVK_UNIT_TESTING_ENABLED
+#include "config.hpp"
+
+// Test for finding file through env.
+// Note : make sure the other paths dont have config files.
+TEST(ConfigTest, FileFromEnvVar) {
+
+    std::string conf_file = "/tmp/tmp-test-config.conf";
```

Comment:
This is not portable across operating systems.
Consider using `std::filesystem::temp_directory_path`